changelog/5961.doc.rst

```diff
@@ -0,0 +1,2 @@
+Added the gallery example :ref:`reprojection_azimuthal_equidistant.py` to illustrate azimuthal equidistant projection.
```

Comment:
This ref is incorrect, you will need to

1. Rename the example to this file name.
2. The way to get a gallery reference is not like this: https://sphinx-gallery.github.io/stable/advanced.html?highlight=reference#cross-referencing